python/ray/rllib/models/multiagentfcnet.py

```diff
@@ -47,7 +47,10 @@ def _init(self, inputs, num_outputs, options):
                         sub_options.update({c: options[c]})
                     if c in custom_options:
                         sub_options.update({c: custom_options[c]})
-                sub_options.update({"fcnet_hiddens": hiddens[i]})
+                if "hierarchical_fcnet_hiddens" in sub_options:
```

Comment:
I think this if-else statement is redundant because the `hierarchical_fcnet_hiddens` or `fcnet_hiddens` should be updated by the preceding for loop, depending on the specified model type.